tools/privacyidea-migrate-linotp.py

```diff
@@ -201,39 +202,47 @@ def migrate(config_obj):
                         Column("rollout_state", Unicode(10), default=u''))
 
     tokenowner_table = Table("tokenowner", metadata,
-                             Column("id", Integer, primary_key=True, nullable=False),
+                             Column("id", Integer, Sequence('tokenowner_seq'),
+                                    primary_key=True, nullable=False),
                              Column("token_id", Integer, ForeignKey("token.id"), nullable=False),
                              Column("resolver", Unicode(120), default=u"", index=True),
                              Column("user_id", Unicode(320), default=u"", index=True),
                              Column("realm_id", Integer, ForeignKey("realm.id"), nullable=False)
                              )
 
     tokeninfo_table = Table("tokeninfo", metadata,
-                            Column("id", Integer, primary_key=True),
+                            Column("id", Integer, Sequence('tokeninfo_seq'),
+                                   primary_key=True),
                             Column("Key", Unicode(255), nullable=False),
                             Column("Value", UnicodeText(), default=u''),
                             Column("Type", Unicode(100), default=u''),
                             Column("Description", Unicode(2000), default=u''),
                             Column("token_id", Integer()))
 
     tokenrealm_table = Table("tokenrealm", metadata,
-                             Column("id", Integer(), primary_key=True),
+                             Column("id", Integer(),
+                                    Sequence('tokenrealm_seq'),
+                                    primary_key=True),
                              Column("token_id", Integer()),
                              Column("realm_id", Integer()))
 
     realm_table = Table("realm", metadata,
-                        Column("id", Integer, primary_key=True),
+                        Column("id", Integer, Sequence('realm_seq'),
+                               primary_key=True),
                         Column("name", Unicode(255), default=u''),
                         Column("default",  Boolean(), default=False),
                         Column("option", Unicode(40), default=u''))
 
     resolver_table = Table("resolver", metadata,
-                           Column("id", Integer, primary_key=True),
+                           Column("id", Integer, Sequence('resolver_seq'),
+                                  primary_key=True),
                            Column("name", Unicode(255), default=u""),
                            Column("rtype", Unicode(255), default=u""))
 
     resolver_config_table = Table("resolverconfig", metadata,
-                                  Column("id", Integer, primary_key=True),
+                                  Column("id", Integer,
+                                         Sequence('resolverconfig_seq'),
```

Comment:
Please change this to "resolverconf_seq", unfortunately this sequence name does not match the table name:
https://github.com/privacyidea/privacyidea/blob/ee2951c18607d82a77461abecd3807d77adc46a4/privacyidea/models.py#L1035

Reply:
Thanks, good pickup.  I have updated the PR.